frontend/src/routes/issue/create/+page.svelte

```diff
@@ -67,11 +67,20 @@
 		params.lat = String(centerPos[0]);
 		params.long = String(centerPos[1]);
 		loadingLocation = true;
-		const res = await libre311.reverseGeocode(centerPos);
-		loadingLocation = false;
-		params.address_string = res.display_name;
-		params = params;
-		goto(linkResolver.createIssuePageNext($page.url));
+
+		try {
+			const res = await libre311.reverseGeocode(centerPos);
+			params.address_string = res.display_name;
+			params = params;
```

Comment:
It seems like lines 74 and 75 can move to the `finally` block which would allow lines 79 and 80 to be deleted.